aiodns/__init__.py

```diff
@@ -6,7 +6,14 @@
     import trollius as asyncio
 import pycares
 
-from . import error
+from typing import (
```

Comment:
Sorry for my ignorance, but why are these needed if we are not really using them (just in comments) ?